cmd/kops/create_cluster.go

```diff
@@ -1078,6 +1078,23 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		}
 	}
 
+	// AllowPrivileged is deprecated and removed in v1.14.
```

Comment:
So I think we can pull these out .... it's a new cluster, it won't / shouldn't have them set.

Reply:
Done